SFPortfolioApp/SFPortfolioApp/Modules/CategoryModule/BaseListScene/ListViewController.swift

```diff
@@ -48,6 +45,14 @@ class ListViewController: BaseViewController {
         }
     }
     
+    private func setupTableView() {
+        categoriesTableView.register(cellNames: viewModel.reuseIdentifiers)
+        let backgroundView = UIImageView(image: UIImage.background)
+        backgroundView.isAccessibilityElement = true
+        backgroundView.accessibilityIdentifier = "listTableViewBackground"
```

Comment:
Same as previous comment

Reply:
The default value is _false_ for a standard UIKit controls and here for the UIView as not a standard control we need to set it as _true_.